# No Title

[Original Post](https://discourse.onlinedegree.iitm.ac.in/t/164277/500)

<p>sir I am noticing a pattern, that when I am running the datagen first. And then using the evaluate.py, then I am getting the A2 right.<br>
But running the evaluation.py for the 2nd time cause the A2 to fail…<br>
Probabbly Because the file in the data folder gets upated should I worry for that…</p>